Source/lua/randomat2/randomat_base.lua

```diff
@@ -282,6 +285,66 @@ end
 
 function randomat_meta:GetConVars() end
 
+-- What role is a player?
+function randomat_meta:GetRoleName(ply)
```

Comment:
Add Phantom and Glitch for the full list

Reply:
Oof, added innocent so it could be used elsewhere but completely forgot those 2.  Done.